### ISSUE TITLE
Add CLI tool

### ISSUE THREAD

**Issue by lgruen** [populationgenomics/analysis-runner#8]
We should have a simple CLI tool that gathers some of the parameters for the request that's sent to the server automatically: e.g. GitHub repository, commit hash. It should also map a human-readable project name to the correct HTTPS endpoint.

**Comment by illusional** [comment]
Hey Leo, just putting together a rough plan for actioning this issue, does this match up with what you were expecting?

A CLI called like:

```
analysis-runner run \
     --output-dir gs://some-path/ \
    [--commit-hash <current-hash>] \
    [--repo <current-repo>]  \
    --script <maybe default: main.py with no args>
```

Process:

1. Fill in the mising info (repo, commit hash, etc)
2. Get the gcloud auth token $(gcloud auth print-identity-token)
3. Form a POST request with params:
    * output
    * repo
    * commit
    * script
    * description
4. Collect and print response

**Comment by illusional** [comment]
Further to this discussion, we need to implement some mechanism for deciding the server. There should be a `--stack` parameter used for determining the server, this is simply looked up in a JSON file hosted on GH. If this tool is used more frequently, we could setup a cloud function to return the correct server.

**Comment by illusional** [comment]
Closed by #12.
